EIPS/eip-x.md

```diff
@@ -0,0 +1,168 @@
+---
+eip: <to be assigned>
+title: Smart Contract Interface for Licences
+author: Lucas Cullen @BitcoinBrisbane, Kai Yeung @CivicKai, Anna Crowley annaelizabethcrowley@gmail.com, Caroline Marshall caroline.marshall888@gmail.com, Katrina Donaghy, katrina@civicledger.com
```

Comment:
Please format these correctly according to EIP-1.

e.g. `Lucas Cullen (@BitcoinBrisbane)` and `Katrina Donaghy <katrina@civicledger.com>`.

Reply:
Fixed

Reply:
Fixed